operations-jms/src/test/java/edu/unc/lib/boxc/operations/jms/streaming/StreamingPropertiesRequestSerializationHelperTest.java

```diff
@@ -21,18 +22,16 @@ public class StreamingPropertiesRequestSerializationHelperTest {
     public void testHelperSuccess() throws IOException {
         var request = new StreamingPropertiesRequest();
         request.setAgent(agent);
-        request.setFilename("banjo_recording.mp3");
         request.setId(pid.getId());
-        request.setFolder(StreamingPropertiesRequest.OPEN);
+        request.setUrl(STREAMREAPER_PREFIX_URL + "?/params=more");
```

Comment:
doesn't matter too much for the test, but we should remove the / in front of params since that isn't a real url pattern